modules/stanford_events_importer/src/stanfordEventsImporter.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace Drupal\stanford_events_importer;
+
+use GuzzleHttp\ClientInterface;
+
+/**
+ * A class to do some importery stuff.
+ */
+class StanfordEventsImporter {
+
+  /**
+   * URL Endpoint for getting categories.
+   */
+  const STANFORD_EVENTS_IMPORTER_XML = "https://events.stanford.edu/xml/drupal/v2.php";
```

Comment:
this is the same as the field widget. maybe we could have a simple interface that each implements.

Reply:
Could be fun to combine the two but not worth the effort at this point IMHO.

Reply:
it could be a really small interface with even just the one const. but not necessary at this time.. was just a thought